### PR TITLE
Add OpenShift 4.8 release notes

### PR DESCRIPTION
**Pull request by tobru**

### CODE REVIEW

**Review by simu — CHANGES_REQUESTED** [review]
Looks like the file `references/release_notes.adoc` is missing

**Comment by tobru** [comment]
> Looks like the file references/release_notes.adoc is missing

Classic case of "How do I git?" :man_facepalming:

File: docs/modules/ROOT/pages/references/release_notes.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+= Release Notes
+
+TIP: This page lists notable changes in OpenShift releases which we find important. Reading release notes for you as a service.
+
+== OpenShift 4.8
+
+OpenShift 4.8 was release at 2021-08-02.
```
Comment (simu) [inline docs/modules/ROOT/pages/references/release_notes.adoc:7]:
```suggestion
OpenShift 4.8 was released on 2021-08-02.
```

**Review by simu — APPROVED** [review]
LGTM overall

**Review by corvus-ch — APPROVED** [review]

**Pull request merged**